problems/cash-units/cash-units.kt

```diff
@@ -0,0 +1,35 @@
+import kotlin.test.assertEquals
+
+private val cashUnits = listOf(500_00, 200_00, 100_00, 50_00, 20_00, 10_00, 5_00, 2_00, 1_00, 50, 20, 10, 5, 2, 1)
```

Comment:
What's the convention for constants in Kotlin? `camlCase` or `SNAKE_CASE`?
Also, the name of the constant is the same as the function name, which might be a bit confusing for newcomers.
Maybe we can rename the constant to avoid ambiguity?

Reply:
`const val` is UPPER_SNAKE_CASE but this is not a `const val`. 😉 I’ll rename it.

Reply:
Actually, why couldn't this be a `const val`? Looks like the cash units are constants for the runtime of the application. 😉

Reply:
Because a list is allocated by the JVM and cannot be made constant. The JVM only allows unboxed primitives and strings to be constant, everything else must go through the JVM memory management.